tests/test_playbooks/content_export_repository.yml

```diff
@@ -0,0 +1,70 @@
+---
+- hosts: localhost
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - include_tasks: tasks/organization.yml
+      vars:
+        organization_state: present
+        organization_name: "Export Org"
+    - include_tasks: tasks/product.yml
+      vars:
+        product_name: "Export Test Product"
+        organization_name: "Export Org"
+        product_state: present
+    - include_tasks: tasks/repository.yml
+      vars:
+        product_name: "Export Test Product"
+        repository_name: "TestRepo"
+        repository_state: present
+        organization_name: "Export Org"
+      register: result
+- hosts: tests
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - include_tasks: tasks/repository.yml
```

Comment:
this looks unnecessary, you've already created the repo in the `localhost` play above